cloud-infrastructure/cluster/main-cluster.bicep

```diff
@@ -12,6 +12,8 @@ param domainName string
 param accountManagementVersion string = ''
 param accountManagementDomainConfigured bool
 param applicationInsightsConnectionString string
+param communicatoinServicesDataLocation string = 'europe'
+param mailSenderDisplayName string = 'PlatformPlatform'
```

Comment:
Should this display name contain "NoReply" or is it only relevant for Azure usage and not externally?

Reply:
It's the display name of the mail that is sent. I investigated all my mails, and I don't see anyone having NoReply in the display name. Most are using no-reply@foo.com (with a dash) while less are using noreply@foo.com.

I've made it configurable, but I don't want yet another GitHub repository variable.